src/vaadin-combo-box.html

```diff
@@ -347,7 +350,14 @@
           return ['_updateAriaExpanded(opened)'];
         }
 
-        attributeChanged(name, type) {
+        /**
+         * @param {string} name
+         * @param {?string} oldValue
+         * @param {?string} newValue
+         * @protected
+         */
+        attributeChangedCallback(name, oldValue, newValue) {
+          super.attributeChangedCallback(name, oldValue, newValue);
```

Comment:
This was changed from legacy `attributeChanged` to `attributeChangedCallback` for consistency. I think it shouldn't affect behaviour and is probably ok if tests pass.